index.d.ts

```diff
@@ -652,7 +652,7 @@ export interface ComboboxProps extends React.ComponentPropsWithoutRef<typeof Box
   /**
    * Properties forwarded to the autocomplete component. Use with caution.
    */
-  autocompleteProps?: AutocompleteProps
+  autocompleteProps?: Omit<AutocompleteProps, 'items' | 'selectedItem' | 'onChange' | 'itemToString' | 'children'>
```

Comment:
Can you describe why we would omit these types from `autocompleteProps`? All of these values are accepted and would in fact take precedent over the top-level props that match.

Reply:
Because `items` is required, so I to use both `autocompleteProps`'s `items` and `ComboboxProps`'s `items`